.automation/kroki.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
This script is quite cool, but I think we would need some way of executing it via GitHub Actions and an update to the .automation README?

Reply:
I've just proposed tools to build architecture diagram, and update descriptors stats in GitHub about repository description and Markdown files, but not all the infrastructure (this is also the main reason ofthe "draft" PR mode)

Reply:
Makes sense. Yeah, let's see what @nvuillam recommends before I waste any of your time chasing geese since I am still ramping up on the overall architecture of MegaLinter.

Reply:
One thing moving to `build.py` gets you for free is that the GitHub Action for that is already wired up, so this comment is moot now.